app.json

```diff
@@ -1,14 +1,9 @@
 {
   "name": "govuk_elements",
+  "stack": "heroku-18",
   "scripts": {
   },
   "env": {
-    "NODE_ENV": {
-      "required": true
-    },
-    "NPM_CONFIG_PRODUCTION": {
-      "required": true
```

Comment:
I suspect this was meant to make the build quicker by only pulling in dependencies required for production. Happy to get rid of both.

Reply:
This makes it _required_, but doesn't actually set it to anything – if we find it's needed, I think we can still set it in the pipeline settings.

Reply:
ah, right. event better, we know they don't help at all at this point.